### ISSUE TITLE
Fix required for Admin 3.7.6+/js-controller 2.2.x

### ISSUE THREAD

**Issue by Apollon77** [sbormann/ioBroker.iqontrol#104]
Please also check for "null" whcn you check the common-custom[namespace] data because with new admin and js-controller < 2.2 it will happen that "null" values can be there instead of "nothing". You currently only check for "undef"

https://github.com/sbormann/ioBroker.iqontrol/blob/master/admin/index_m.js#L1286

**Comment by sbormann** [comment]
Hi, wich data exactly can be null?:
adapter.common.custom
or
adapter.common.custom.namespace?

**Comment by Apollon77** [comment]
In fact common.custom.namespace ... common.custom should be an object or undefined. 

In general: why you check that derailed on undef ? Just a „not“ should be enough and cover null and undefined (ok and 0 ...)

**Comment by sbormann** [comment]
This is a result of try-n-error and missing knowledge. I always asked my self, if there was a better way to do it. 

Could you please help me and give an example, how you would express this line in a better way:
``
if (typeof usedObjects[linkedStateId].common.custom !== udef 
&& typeof usedObjects[linkedStateId].common.custom[namespace] !== udef 
&& usedObjects[linkedStateId].common.custom[namespace] !== null
&& typeof usedObjects[linkedStateId].common.custom[namespace].readonly !== udef) result = usedObjects[linkedStateId].common.custom[namespace].readonly;
``
Many thanks!

**Comment by Apollon77** [comment]
In fact: if the value can not be 0 oder false (so mostly do that as soon as an object should be there or not you can simply do:

```
if (usedObjects[linkedStateId].common.custom && usedObjects[linkedStateId].common.custom[namespace] && usedObjects[linkedStateId].common.custom[namespace].readonly !== undefined) result = usedObjects[linkedStateId].common.custom[namespace].readonly;
```

**Comment by Apollon77** [comment]
A check for "usedObjects[linkedStateId].common.custom" checks if the value is
* not null
* not undefined
* not 0 (evaluates to false)
* not false

**Comment by sbormann** [comment]
Should be fixed in 0.2.15
